keepercommander/importer/keepass/keepass.py

```diff
@@ -280,9 +280,11 @@ def extension(self):
 
 class KeepassExporter(BaseExporter):
 
-    def do_export(self, filename, records):
-        print('Choose password for your Keepass file')
-        master_password = getpass.getpass(prompt='...' + 'Keepass Password'.rjust(20) + ': ', stream=None)
+    def do_export(self, filename, records, file_password):
```

Comment:
Looks like this function overloads the existing `do_export` in the parent class `BaseExporter`. maybe set it to `None` by default in the base class?